spec/requests/registrations_spec.rb

```diff
@@ -15,27 +15,36 @@
         end
       end
 
-      it "shows the sign in text for password based authentication" do
+      it "only shows the single sign on options if they are present" do
+        allow(Authentication::Providers).to receive(:enabled).and_return([])
+
         get sign_up_path
 
-        expect(response.body).to include("Have a password? Continue with your email address")
+        expect(response.body).not_to include("Continue with")
```

Comment:
```suggestion
        expect(response.body).not_to include("Have a password? Continue with your email address")
```
Any reason we can't include the full text in this assertion? Just from reading the test, expecting `"Continue with"` to _not_ render doesn't make it entirely clear what the behavior should be, IMO.